domains/manufacturing-environments/discover-behavior-specifications/q5.csv

```diff
@@ -0,0 +1,3 @@
+form,
+http://example.org/coapForm>,
```

Comment:
In addition to the extra comma at the end of the line as mentioned in a previous comment, there is an additional `>` character in this line

Reply:
Addressed in a past commit.